app/forms/contact_form.rb

```diff
@@ -0,0 +1,9 @@
+class ContactForm
+  include ActiveModel::Model
+
+  attr_accessor :want_learn, :want_tell, :email_or_tel, :comments
+
+  def submit(_)
+    valid?
```

Comment:
This will always return true since there aren't any validations defined. If we don't need to validate any part of the contact form, then we don't need a form object at all. The controller can call `UserMailer.contact_request(form_params).deliver_later` directly without any other logic.

If we do want to validate one or more of the fields in the form, then we need to add validations to this form object.